docs/housing_search.md

```diff
@@ -24,6 +24,15 @@ ElasticSearch will be updated by a Lambda function, holding business logic, that
 ** Search by : **
 1. Person - First name, Middle name, Last name
 2. Asset - Address line 1, Postcode, Asset type
+3. Tenure - Payment Reference, FullAddress of TenuredAsset , Household Members FullName  
+4. Transactions - Person FullName, Transaction Type, Payment Reference, Bank Account Number, Transaction Date, Transaction Amount  
+
+### Considerations
+#### Transactions
+* The search must be flexible enough to allow for the successful finding of information without assuming the form of the query. An officer may know the PRN, or the address, or just the name.
```

Comment:
Hi Hanna, do you mind adding the point here about the use case for possibly searching for a transaction by the bank account number (the use case Phillip described) and specifying that by name, we mean name of sender (person or company)

Reply:
Added more info here